tests/test_models.py

```diff
@@ -871,6 +871,38 @@ def test_string_conversion(self, method):
         )
         self.assertEqual(method(enterprise_catalog), expected_str)
 
+    @mock.patch('enterprise.utils.configuration_helpers')
+    def test_catalog_info_in_course_enrollment_url(self, config_mock):
+        faker = FakerFactory.create()
```

Comment:
create a setUp method for this test class and move these 3 common line there to avoid repetition:
```python
faker = FakerFactory.create()
uuid = faker.uuid4()  # pylint: disable=no-member
name = 'EnterpriseWithACatalog'
```